solutions/cses-1074.mdx

```diff
@@ -48,3 +53,41 @@ int main(){
 	return 0;
 }
 ```
+</CPPSection>
+
+<JavaSection>
+
+```java
+
+import java.util.*;
+import java.io.*;
+
+public class stickLengths {
+CodeSnip{Kattio}
+public static void main(String[] args) {
+	Kattio io = new Kattio();
+	int N = io.nextInt();
+	int[] arr = new int[N];
+	for (int i = 0; i < N; i++) {
+		arr[i] = io.nextInt();
+	}
+	Arrays.sort(arr);
+
+	/**
+	 * Take the median stick length,
+	 * and calculate the cost to change each stick length to the median.
+	 */
+	long median = arr[N / 2];
+		long costs = 0;
```

Comment:
```suggestion
	long costs = 0;
```

Reply:
it's not this part that has the wrong indentation, the entire main function's indentation is wrong